### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by raq929**
Adds a checklist for PRs, basically identical to the one on `edx-enterprise`

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - *Note*: This **must** be run if you modified any models.
+      - It may or may not make a migration depending on exactly what you modified, but it should still be run.
+    - This should be run from either a venv with all the lms/edx-enterprise requirements installed or if you checked out edx-enterprise into the src directory used by lms, you can run this command through an lms shell.
+        - It would be `./manage.py lms makemigrations` in the shell.
+- [ ] [Version](https://github.com/edx/edx-enterprise-data/blob/master/enterprise/__init__.py) bumped
+- [ ] [Changelog](https://github.com/edx/edx-enterprise-data/blob/master/CHANGELOG.rst) record added
+- [ ] Translations updated (see docs/internationalization.rst but also this isn't blocking for merge atm)
+
+**Post merge:**
+- [ ] Tag pushed and a new [version](https://github.com/edx/edx-enterprise/releases) released
```
Comment (iloveagent57) [inline .github/pull_request_template.md]: I think you want `edx-enterprise-data/releases` here, right?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    - *Note*: Assets will be added automatically. You just need to provide a tag (should match your version number) and title and description.
+- [ ] After versioned build finishes in [Travis](https://travis-ci.org/github/edx/edx-enterprise), verify version has been pushed to [PyPI](https://pypi.org/project/edx-enterprise-data/)
```
Comment (iloveagent57) [inline .github/pull_request_template.md]: `github/edx/edx-enterprise-data`

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    - Each step in the release build has a condition flag that checks if the rest of the steps are done and if so will deploy to PyPi.
+    (so basically once your build finishes, after maybe a minute you should see the new version in PyPi automatically (on refresh))
+- [ ] PR created in [edx-analytics-data-api](https://github.com/edx/edx-analytics-data-api) to upgrade dependencies (including edx-enterprise-data)
+    - This **must** be done after the version is visible in PyPi as `make upgrade` in edx-platform will look for the latest version in PyPi.
```
Comment (iloveagent57) [inline .github/pull_request_template.md]: do you mean `edx-analytics-data-api` here, and not `edx-platform`?

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    - Note: the edx-enterprise constraint in edx-platform **must** also be bumped to the latest version in PyPi.
```
Comment (iloveagent57) [inline .github/pull_request_template.md]: "edx-enterprise_data" and "edx-analytics-data-api"?

**Review by iloveagent57 — COMMENTED** [review]
Couple repo names to correct (I think - but I don't have any experience with this repo or the data API, so maybe _I'm_ the one who's confused about what's installed, where).

**Comment by raq929** [comment]
No, you're right, I just followed github instructions to make the template, which were like, just create the file on githib! So I didn't have find and replace. :)

**Review by iloveagent57 — APPROVED** [review]

**Pull request merged**